src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -42,13 +42,13 @@
 import java.util.TreeSet;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 import net.sf.jabref.event.source.EntryEventSource;
 import net.sf.jabref.model.entry.BibEntry;
 import net.sf.jabref.model.entry.BibtexString;
 import net.sf.jabref.model.entry.EntryUtil;
 import net.sf.jabref.model.entry.FieldName;
-import net.sf.jabref.model.entry.InternalBibtexFields;
```

Comment:
Why do you remove the import of that class? - You did not say it in your comments https://github.com/JabRef/jabref/pull/1577#issue-165098695